students/arun_nalla/session08/circle.py

```diff
@@ -0,0 +1,120 @@
+#! usr/bin/env phtyon
+
+""" Circle class assignment by
+Arun Nalla 12/08/2018"""
+
+from math import pi
+import random
+
+class Circle():
+
+    def __init__(self, radius):
+        if radius >= 1:
```

Comment:
Shouldn't this be `if radius > 0:`, if you don't want to allow 0-radius circle? Note that radius doesn't have to be integer. It's a minor point, though, I suppose...

Reply:
Thanks Housung, added revised file.